drivers/display/display_otm8009a.c

```diff
@@ -97,7 +97,7 @@ static int otm8009a_configure(const struct device *dev)
 	uint8_t buf[4];
 	int ret;
 
-	static const uint8_t pwr_ctrl2[] = {0x96, 0x34, 0x01, 0x33, 0x33, 0x34, 0x33};
+	static const uint8_t pwr_ctrl2[] = {0x96, 0x34, 0x01, 0x33, 0x33, 0x34, 0x43};
```

Comment:
Does it have an impact on the comment made when applying this config ?
```
	/* power control settings 2 for normal mode */
	/* set pump 4 vgh voltage from 15.0v down to 13.0v */
	/* set pump 5 vgh voltage from -12.0v downto -9.0v */
	/* set pump 4&5 x6 (ONLY VALID when PUMP4_EN_ASDM_HV = "0") */
	/* change pump4 clock ratio from 1 line to 1/2 line */
```

Reply:
Below copy from otm8009a datasheet. `0x43` is reset value for 151 parameter.
 The comment not realted to 151 parameter.

![Screenshot from 2023-12-18 18-23-12](https://github.com/zephyrproject-rtos/zephyr/assets/40390867/d7f44d92-5ccc-4414-860a-79ba3740764d)

Reply:
no impact on the comment